doc/cug.tex

```diff
@@ -3720,6 +3720,34 @@ \subsubsection{Trigger Types}
 and at the first \lstinline=T12= cycle point. Thereafter, \lstinline=foo= just
 depends on its previous (12 hours ago) instance.
 
+Finally, it is also possible to have a dependency on a task at a specific cycle
+point.
+
+\lstset{language=suiterc}
+\begin{lstlisting}
+[scheduling]
+    [[dependencies]]
+        [[[R1/20200202]]]
+            graph = "baz[20200101] => qux"
+\end{lstlisting}
+\lstset{language=transcript}
+
+However, a repeating cycle should not have a dependency on a task wtih a
+specific cycle point. In the following bad example, all instances of
+\lstinline=qux= will depend on \lstinline=baz.20200101=, which will never be
+removed from the task pool:
+
+\lstset{language=suiterc}
+\begin{lstlisting}
+[scheduling]
+    initial cycle point = 2010
+    [[dependencies]]
+        # This is BAD!
+        [[[P1D]]]
+            graph = "baz[20200101] => qux"
+\end{lstlisting}
+\lstset{language=transcript}
+
```

Comment:
Yes, this is the same as `baz[^] => qux` in a cycling section (see #1773). I'll explain the performance issue in the docs for now.